ivy_tests/test_ivy/test_frontends/test_mindspore/test_ops/test_function/test_mindspore_nn_func.py

```diff
@@ -360,3 +360,37 @@
 #         count_include_pad=count_include_pad,
 #         divisor_override=None,
 #     )
+
+# def _filter_dtypes(input_dtype):
+#     assume(("bfloat16" not in input_dtype) and ("float16" not in input_dtype))
+
+# # softshrink
+# @handle_frontend_test(
+#     fn_tree="mindspore.ops.function.nn_func.softshrink",
+#     dtype_and_input=helpers.dtype_and_values(
+#         available_dtypes=helpers.get_dtypes("float"),
+#     ),
+#     lambd=helpers.floats(min_value=0, max_value=1, exclude_min=True),
+# )
+# def test_torch_softshrink(
+#     *,
+#     dtype_and_input,
+#     lambd,
+#     on_device,
+#     fn_tree,
+#     frontend,
+#     test_flags,
+#     backend_fw,
+# ):
+#     input_dtype, x = dtype_and_input
+#     _filter_dtypes(input_dtype)
+#     helpers.test_frontend_function(
+#         input_dtypes=input_dtype,
+#         backend_to_test=backend_fw,
+#         frontend=frontend,
+#         test_flags=test_flags,
+#         fn_tree=fn_tree,
+#         on_device=on_device,
+#         x=x[0],
+#         lambd=lambd,
+#     )
```

Comment:
why did you comment this out?

Reply:
@a0m0rajab there is no backend support yet for mindspore... 
So we have commented out the code